release-checks.sh

```diff
@@ -57,6 +60,16 @@ then
     exit 1
 fi
 
+if [[ $(uname) == "Darwin" ]]; then
```

Comment:
I think this check should be move to precondition checks for the integration tests (because this is where this matters as far as I understand) but not to the precondition checks of the release?

Reply:
https://github.com/elastic/rally/pull/914/commits/601eb0ebb08c31769662843d1cd5b3b9adde2c9f